src/Microsoft.Azure.SignalR.Common/Constants.cs

```diff
@@ -10,6 +10,7 @@ internal static class Constants
         public const string ServerStickyModeDefaultKey = "Azure:SignalR:ServerStickyMode";
         public const string ConnectionStringDefaultKey = "Azure:SignalR:ConnectionString";
         public const string ApplicationNameDefaultKey = "Azure:SignalR:ApplicationName";
+        public const string EndpointsDefaultKey = "Azure:SignalR:Endpoints";
```

Comment:
>Endpoints [](start = 65, length = 9)

`ConnectionString` is reused for `Endpoints` already

Reply:
Yes, you're right. Didn't notice it worked for multiple endpoints. Will revert this.